content/Simple dynamic settings for Django - Jorge Michelena.md

```diff
@@ -0,0 +1,100 @@
+## Simple dynamic settings for Django
+
+
+### Introduction
+
+Not long ago I finished working on a small project in Django, and I noted that there were a few functionalities depending directly on constant values defined in the *settings* file but that given the nature of said functionalities it seemed like a good idea to be able to modify those values to alter the application behaviour without having to edit the settings and deploy the project again.
+
+After doing some research on the topic I learned that there were a few libraries that provided an implementation to handle editable or dynamic setting, but I felt they were a little overkill for the purposes of this project. Given that, I decided to implement my own solution wich is simpler than the ones already implemented.
+
+
+### Editing periodic celery tasks schedules
+
+The project included a couple periodic tasks running with Celery whose schedules I wanted to be able to edit, in this case the solution was very straightforward thanks to [django-celery-beats](https://github.com/celery/django-celery-beat).
+
+Django-celery-beats is a library for Django that provides us with models for periodic tasks and models that helps us define and modify with a great degree of freedom on when and how those tasks will be executed and it can be done through code or through the Django admin page, which results very helpful.
```

Comment:
```suggestion
Django-celery-beat is a library for Django that provides us with models for periodic tasks and models that helps us define and modify with a great degree of freedom when and how those tasks will be executed. It can be done through code or the Django admin page, which results very helpful.
```